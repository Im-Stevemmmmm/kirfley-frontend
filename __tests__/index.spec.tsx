import { MockedProvider } from "@apollo/client/testing";
import { render } from "@testing-library/react";
import Index from "pages/index";

describe("index page", () => {
  it("renders correctly", () => {
    const { container } = render(
      <MockedProvider>
        <Index />
      </MockedProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
