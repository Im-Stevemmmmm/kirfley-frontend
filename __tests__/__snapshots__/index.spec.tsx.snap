// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page renders correctly 1`] = `
<div>
  <div>
    <title>
      Kirfley
    </title>
    <main>
      <nav
        class="navbar"
      >
        <div
          class="navbar__icon"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,<svg width=\\"35\\" height=\\"35\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                style="max-width: 100%; display: block;"
              />
            </div>
            <img
              alt="Navbar Icon"
              decoding="async"
              src="/_next/image?url=%2Ficons%2Fnav-icon.svg&w=320&q=75"
              srcset="/_next/image?url=%2Ficons%2Fnav-icon.svg&w=320&q=75 1x"
              style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <ul
          class="navbar__navItems"
        >
          <li
            class="navbar__nav"
          >
            Support
          </li>
          <li
            class="navbar__nav"
          >
            Log In
          </li>
          <li
            class="navbar__nav"
          >
            Sign Up
          </li>
        </ul>
      </nav>
      <div
        class="banner"
      >
        <div
          class="banner__child"
        >
          <h1
            class="banner__title"
          >
            Kirfley
          </h1>
          <h2>
            Share Your Opinions With the World
          </h2>
          <div
            class="banner__buttons"
          >
            <a
              class="inline buttons__button"
            >
              Log In
            </a>
            <a
              class="inline buttons__button"
            >
              Create an Account
            </a>
          </div>
        </div>
        <div
          class="banner__child"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;charset=utf-8,<svg width=\\"400\\" height=\\"600\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                style="max-width: 100%; display: block;"
              />
            </div>
            <img
              alt=""
              decoding="async"
              src="/_next/image?url=%2Fart%2Findex%2Fphone-taking-image.svg&w=1200&q=75"
              srcset="/_next/image?url=%2Fart%2Findex%2Fphone-taking-image.svg&w=420&q=75 1x, /_next/image?url=%2Fart%2Findex%2Fphone-taking-image.svg&w=1024&q=75 2x, /_next/image?url=%2Fart%2Findex%2Fphone-taking-image.svg&w=1200&q=75 3x"
              style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
